packages/x-tree-view/src/internals/plugins/defaultPlugins.ts

```diff
@@ -0,0 +1,29 @@
+import { useTreeViewNodes, UseTreeViewNodesParameters } from './useTreeViewNodes';
+import { useTreeViewExpansion, UseTreeViewExpansionParameters } from './useTreeViewExpansion';
+import { useTreeViewSelection, UseTreeViewSelectionParameters } from './useTreeViewSelection';
+import { useTreeViewFocus, UseTreeViewFocusParameters } from './useTreeViewFocus';
+import { useTreeViewKeyboardNavigation } from './useTreeViewKeyboardNavigation';
+import {
+  useTreeViewContextValueBuilder,
+  UseTreeViewContextValueBuilderParameters,
+} from './useTreeViewContextValueBuilder';
+import { ConvertPluginsIntoSignatures } from '../models';
+
+export const DEFAULT_TREE_VIEW_PLUGINS = [
```

Comment:
A follow up would be to remove from this list the plugins that are mandatory for the TreeView to work (`useTreeViewNodes` and `useTreeViewContextBuilder` are the only one I think.